contracts/LendingPool.sol

```diff
@@ -188,9 +199,9 @@ contract LendingPool is OwnableUpgradeable, ERC721Upgradeable, Clone {
         uint length = loansToRepay.length;
         uint totalToRepay = 0;
         uint i = 0;
-        while(i<length){
-            totalToRepay += _repay(loansToRepay[i], from);
+        while(i<length) {
             unchecked {
+                totalToRepay += _repay(loansToRepay[i], from);
```

Comment:
llama needs explanation

Reply:
totalToRepay would have to be over 1.15 * 10^59 ETH to overflow

Reply:
interest can be made arbitrary so it could be?